collect_app/src/main/res/xml/form_entry_access_preferences.xml

```diff
@@ -40,8 +40,15 @@
             android:title="@string/save_as_draft"
             app:iconSpaceReserved="false" />
         <CheckBoxPreference
-            android:key="finalize"
+            android:key="finalize_in_form_entry"
             android:title="@string/finalize"
             app:iconSpaceReserved="false" />
     </PreferenceCategory>
-</PreferenceScreen>
\ No newline at end of file
+
+    <PreferenceCategory android:title="@string/uncheck_to_hide_from_drafts">
```

Comment:
I think this should have `app:iconSpaceReserved="false"` otherwise there is empty space before the title.